regression-tests.dnsdist/test_EDE.py

```diff
@@ -104,7 +104,7 @@ def testExtendedErrorBackendResponse(self):
             (_, receivedResponse) = sender(query, response=None, useQueue=False)
             self.checkMessageEDNS(expectedResponse, receivedResponse)
 
-    def testExtendedErrorBackendResponse(self):
+    def testExtendedErrorBackendResponseDO(self):
```

Comment:
there were a couple of test functions w/ multiple definitions, I believe that means only the last one was actually tested. I've tried to rename things in the spirit of what they seemed to be testing.